keps/sig-instrumentation/2305-metrics-cardinality-enforcement/kep.yaml

```diff
@@ -15,9 +16,10 @@ reviewers:
 approvers:
   - "@ehashman"
 creation-date: 2020-04-15
-last-updated: 2021-02-08
-stage: alpha
-status: implementable
-latest-milestone: "v1.21"
+last-updated: 2023-05-28
+stage: beta
```

Comment:
If you're graduating to Beta, you need to also update prod-readiness:
https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-instrumentation/2305.yaml

Not sure why validation didn't catch this...

Reply:
Actually validation caught that, verify is failing for that reason.

Reply:
Thank you for pointing that out, @wojtek-t. I'm not sure whom I should put down here: https://github.com/rexagod/enhancements/blob/c4bb8708d40dd7bb6868f38ac3fcebdf60b2e91f/keps/prod-readiness/sig-instrumentation/2305.yaml from the PRR team (I think Elana is unavailable).

Reply:
Should I check with the PRR folks (including you) for their interest in being an approver for the beta graduation?

Reply:
I'm happy to take it - feel free to put me.